apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx

```diff
@@ -42,7 +45,13 @@ export const BreakpointsSelector = () => {
             return (
               <ToolbarToggleItem
                 variant="subtle"
-                ref={breakpoint.id === selectedBreakpoint.id ? ref : undefined}
+                ref={(node) => {
+                  if (node) {
+                    refs.current.set(breakpoint.id, node);
+                    return;
+                  }
+                  refs.current.delete(breakpoint.id);
+                }}
```

Comment:
I think ref should be stable otherwise remounts on every render. But I'm not sure

Reply:
I copypasted from react https://react.dev/learn/manipulating-the-dom-with-refs#how-to-manage-a-list-of-refs-using-a-ref-callback and here are elements not refs
So map is not stable but map values are

Reply:
I don't think ref needs to be stable

Reply:
Ah got it. It does not remount, just recall again

> React will also call your ref callback whenever you pass a different ref callback. In the above example, (node) => { ... } is a different function on every render. When your component re-renders, the previous function will be called with null as the argument, and the next function will be called with the DOM node.